### PR TITLE
added page(s) to 341 line

### PR DESCRIPTION
**Pull request by MuhammadHasnain77**

### CODE REVIEW

**Comment by MuhammadHasnain77** [comment]
is it correct?
I am new to github so I don't know much about it.

**Review by pmeier — CHANGES_REQUESTED** [review]
Hey @MuhammadHasnain77 and welcome to the Ragna project :wave: Unfortunately, it is not in the right place or better not in the right place I wanted it to have. Let me add a screenshot to the issue to get a better sense.

File: ragna/deploy/_ui/central_view.py [hunk 1]
```diff
@@ -336,9 +336,16 @@ def on_click_source_info_wrapper(self, event, sources):
         ]
         for rank, source in enumerate(sources, 1):
             location = source["location"]
+            pages = source["document"].get("pages")
```
Comment (pmeier) [inline ragna/deploy/_ui/central_view.py:339]: `source["document"]` does not have a field called `pages`. You can find the schemas here

https://github.com/Quansight/ragna/blob/38f75f6d88e74f72c030ef50045315b81eb203aa/ragna/deploy/_api/schemas.py#L35-L39

https://github.com/Quansight/ragna/blob/38f75f6d88e74f72c030ef50045315b81eb203aa/ragna/deploy/_api/schemas.py#L18-L20

The `location` field is what you want. You can check if it is a non-empty string and if yes, prepend `page(s)` to it. You can use the following existing block for it:

https://github.com/Quansight/ragna/blob/38f75f6d88e74f72c030ef50045315b81eb203aa/ragna/deploy/_ui/central_view.py#L339-L340

**Comment by pmeier** [comment]
@MuhammadHasnain77 we ended up touching the relevant lines in #264 and fixed #272 in the process. Thus, I'm closing this PR. Still, thanks for the effort.

**Pull request closed without merging**